utils/build-script-impl

```diff
@@ -2656,10 +2656,10 @@ for host in "${ALL_HOSTS[@]}"; do
                 fi
                 ;;
             llvm)
-                if [[ -z "${LLVM_INSTALL_COMPONENTS}" ]] ; then
+                if [[ -z "${LLVM_INSTALL_COMPONENTS}" ]] && [[ -z "${INSTALL_LLVM}" ]]; then
                     continue
                 fi
-                if [[ "${LLVM_INSTALL_COMPONENTS}" == "all" ]]; then
+                if [[ "${LLVM_INSTALL_COMPONENTS}" == "all" ]] || [[ -n "${INSTALL_LLVM}" ]] ; then
```

Comment:
How does this compares to what our bots are using, which seems to be usually
```
llvm-cov;llvm-profdata;IndexStore;clang;clang-resource-headers;compiler-rt;clangd;
  lld (linux)
  dsymutil (darwin)
```

Does "all" include installing e.g. llvm and clang development libraries?  If so, that is probably not a good default.